packages/teleport/src/useWebSession.ts

```diff
@@ -0,0 +1,28 @@
+/*
+Copyright 2022 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import React from 'react';
+
+import { SessionContext } from './WebSessionContext';
+
+export default function useWebSession() {
+  const sessionContext = React.useContext(SessionContext);
+  if (!sessionContext) {
+    throw new Error('Unable to retrieve WebSession Context');
+  }
+
+  return (window['sessionContext'] = sessionContext);
```

Comment:
Does this need to be on the window?

Reply:
Good catch. I was using this to make it easier to debug from the console and forgot to remove it.